test/models/user_test.rb

```diff
@@ -3,6 +3,34 @@
 require "test_helper"
 
 class UserTest < ActiveSupport::TestCase
+  context "avatarable" do
```

Comment:
Should I move this into a test helper concern?

Reply:
Yes I think so!

Reply:
done!